docs/migration/cosmoshub-3.md

```diff
@@ -1,21 +1,47 @@
+The following document describes the necessary steps involved that validators and full node operators
+must take in order to upgrade from `cosmoshub-3` to `cosmoshub-4`. The Cosmos teams
+will post an official `cosmoshub-4` genesis file, but it is recommended that validators
 execute the following instructions in order to verify the resulting genesis file.
 
 There is a strong social consensus around proposal `Cosmos Hub 4 Upgrade Proposal`
 on `cosmoshub-3`. Following proposals #[27](https://www.mintscan.io/cosmos/proposals/27), #[35](https://www.mintscan.io/cosmos/proposals/35) and #[36](https://www.mintscan.io/cosmos/proposals/36).
 This indicates that the upgrade procedure should be performed on `February 18, 2021 at 06:00 UTC`.
 
+  - [Summary](#summary)
   - [Migrations](#migrations)
   - [Preliminary](#preliminary)
   - [Major Updates](#major-updates)
   - [Risks](#risks)
   - [Recovery](#recovery)
   - [Upgrade Procedure](#upgrade-procedure)
+  - [Guidance for Full Node Operators](#guidance-for-full-node-operators)
   - [Notes for Service Providers](#notes-for-service-providers)
+ 
+# Summary
+
+The Cosmoshub-3 will undergo a scheduled upgrade to Cosmoshub-4 on Feb 18, 2021 at 6 UTC.
+
+The following is a short summary of the upgrade steps:
+    1. Stopping the running Gaia v2.0.x instance 
+    1. Backing up configs, data, and keys used for running Cosmoshub-3
+    1. Resetting state to clear the local Cosmoshub-3 state
+    1. Copying the cosmoshub-4 genesis file to the Gaia config folder (either after migrating an existing cosmoshub-3 genesis export, or downloading the cosmoshub-4 genesis from the mainnet github)
+    1. Installing the Gaia v4.0.x release
+    1. Starting the Gaia v4.0.x instance to resume the Cosmos hub chain at a height of <cosmoshub3 height> + 1.
+
+Specific instructions for validators are available in [Upgrade Procedure](#upgrade-procedure), 
+and specific instructions for full node operators are available in [Guidance for Full Node Operators](#guidance-for-full-node-operators).
+
+Upgrade coordination and support for validators will be available on the #validators-verified channel of the [Cosmos Discord](https://discord.gg/vcExX9T).
+
+The network upgrade can take the following potential pathways:
+1. Happy path: Validator successfully migrate the cosmoshub-3 genesis file to a cosmoshub-4 genesis file, and the validator can succesfully start Gaia v4 with the cosmoshub-4 genesis within 1-2 hours of the scheduled upgrade.
+1. Not-so-happy path: Validators have trouble migrating the cosmoshub-3 genesis to a cosmoshub-4 genesis, but can obtain the genesis file from the Cosmos mainnet github repo and can successfully start Gaia v4 within 1-2 hours of the scheduled upgrade.  
+1. Abort path: In the rare event that the team becomes aware of critical issues, which result in an unsuccessful migration within a few hours, the upgrade will be announced as aborted 
```

Comment:
```suggestion
3. Abort path: In the rare event that the team becomes aware of critical issues, which result in an unsuccessful migration within a few hours, the upgrade will be announced as aborted 
```

Reply:
Markdown formats every point automatically incrementally when always using `1` - this way you do not have to worry to keep items aligned in case something changes.